### PR TITLE
29 general task parents

### PR DESCRIPTION
**Pull request by chrisbc**
- fix (With test ) for https://github.com/GNS-Science/nshm-toshi-api/issues/29
- This will PR will push changes (via github actions) to our AWS test environment. 
   We're just getting this API ready for use in PROD v soon.

### CODE REVIEW

**Comment by chrisbc** [comment]
@wooneh I need to do some more work on the docs for getting this project running. 

short form:
 - install the serverless (sls) framework (uses Node)
     this is needed even for local testing
 - run a local elastic search instance:
     `docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:6.8.0`
 - run a local S3 service:
    `sls s3 start` -  it will write S3 objects to a `\tmp\nzshm22*` folder
 - run the  API service locally:
     `sls wsgi serve`  - by default serves up http:\\localhost:5000\graphql
- run some tests:
    ```
    source bin/activate
    pytest
    python graphql_api/tests/smoketests.py
    ```

**Comment by wooneh** [comment]
Cheers, I've gone ahead and got it running. Had to add missing dependency in Flask-Cors, should be added to `requirements.txt`.

File: graphql_api/data_s3/base_s3_data.py [hunk 1]
```diff
@@ -44,16 +45,23 @@ def get_one(self, _id):
         """
         raise NotImplementedError("method needs to be defined by sub-class")
 
-    def get_all(self):
+    def get_all(self, clazz_name=None):
```
Comment (wooneh) [inline graphql_api/data_s3/base_s3_data.py:48]: One of the tests mocks this function, which fails due to the updated function parameter:

`graphql_api\tests\test_schema.py:122: TypeError`
`{'errors': [{'message': '<lambda>() takes 1 positional argument but 2 were given', 'locations': [{'line': 2, 'column': 21}], 'path': ['rupture_generation_tasks']}], 'data': {'rupture_generation_tasks': None}}`

Would need to update the lambda used by the mock.

Comment (chrisbc) [reply graphql_api/data_s3/base_s3_data.py:48]: Right - i have fixed this already, but on another branch! Ooops.

**Comment by chrisbc** [comment]
> Cheers, I've gone ahead and got it running. Had to add missing dependency in Flask-Cors, should be added to `requirements.txt`.

Oh thanks I'll add this.

**Pull request merged**